app/containers/message/interfaces.ts

```diff
@@ -7,7 +7,7 @@ export type TMessageType = 'discussion-created' | 'jitsi_call_started';
 
 export interface IMessageAttachments {
 	attachments: any;
-	timeFormat: string;
+	timeFormat?: string;
```

Comment:
[NON-BLOCKING]: Now that we have our definitions, I was wondering if it would be better to use the message definitions from the definitions folder, instead of using these interfaces.

e.g: file: `app/containers/message/Discussion.tsx`

Before:
```
...

const Discussion = React.memo(
	({ msg, dcount, dlm, theme }: IMessageDiscussion) => {
		let time;
... 
```

After:
```
...

const Discussion = React.memo(
	({ msg, dcount, dlm, theme }: Pick<IMessage, 'msg' | 'dcount' | 'dlm'> & { theme: string }) => {
		let time;
... 
```